source/users.markdown

```diff
@@ -229,5 +245,4 @@ If you noticed something wrong  with your entry here (or if you’d like us to r
 * Xtendsys
 * Zajil Telecom
 * Zenoss
-* Zynga 
-
+* Zynga
```

Comment:
I have no idea why git thinks there is a change here, but the end result remain the same